CHANGELOG.md

```diff
@@ -8,10 +8,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- **CUMULUS-2475**
-  - Adds `GET` endpoint to distribution API
-- **CUMULUS-2476**
-  - Adds handler for authenticated `HEAD` Distribution requests replicating current behavior of TEA 
 - **CUMULUS-2478**
   - Implemented [bucket map](https://github.com/asfadmin/thin-egress-app#bucket-mapping).
```

Comment:
The bucket map stuff isn't in this release?

Reply:
Yeah, that's right. I merged master (with the bucket map stuff) in after cutting the release